### ISSUE TITLE
Faster reaction while ticking of items

### ISSUE THREAD

**Issue by jeena** [CampDev/Tasky#39]
The icon should change same time as I click on "done", then you can send a async ajax request to change it on the server. Otherwise waiting for the server takes 1-2 seconds which makes the UI feel really slow.

Btw. the same with creating new items, you shouldn't reload the whole page, just put the new task in with JavaScript and asynchronously send it back to the server.

**Comment by Nygu** [comment]
I agree that implementing ajax wouldn't hurt the user experience. It may hurt the developer experience though. :p 

Lets see how Yannik feels about this. :)

**Comment by Cacauu** [comment]
You're right, that's definitely a thing to consider and something that should be changed in the future. Ajax sounds like the best solution for that at the moment, my problem is just that I never wrote any Ajax code and it will take me some days or weeks to get familiar with the language. I'll start with that as soon as possible and will try to implement it into Tasky soon.

In case anyone of you has any good Ajax tutorials or reads, please let me know and send me links.
